internal/convert.go

```diff
@@ -345,9 +345,9 @@ func (conv *Conv) AddPrimaryKeys() {
 			}
 			if !primaryKeyPopulated {
 				k := conv.buildPrimaryKey(t)
-				ct.ColNames = append(ct.ColNames, k)
+				ct.ColIds = append(ct.ColIds, k)
 				ct.ColDefs[k] = ddl.ColumnDef{Name: k, T: ddl.Type{Name: ddl.String, Len: 50}}
-				ct.Pks = []ddl.IndexKey{{Col: k}}
+				ct.PrimaryKeys = []ddl.IndexKey{{ColId: k}}
```

Comment:
Why are we assigning k - column name to colID ?

Reply:
Added id to synthetic primary key.